bridge/server/app.ts

```diff
@@ -26,7 +26,6 @@ const __dirname = dirname(fileURLToPath(import.meta.url));
 const apiUrl: string | undefined = process.env.API_URL;
 let apiToken: string | undefined = process.env.API_TOKEN;
 let cliDownloadLink: string | undefined = process.env.CLI_DOWNLOAD_LINK;
-let integrationsPageLink: string | undefined = process.env.INTEGRATIONS_PAGE_LINK;
```

Comment:
With this being removed, could you also remove the other usages? Else there will be a compile error

Reply:
> where I can find the other usages

It's right below:
https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/bridge/server/app.ts#L82-L85